detox/src/artifacts/ArtifactsManager.js

```diff
@@ -111,6 +112,10 @@ class ArtifactsManager {
     await this._callPlugins('plain', 'onLaunchApp', appLaunchInfo);
   }
 
+  async onAppReady(appInfo) {
```

Comment:
This way I can tracking if the app launch was not successful and, for instance, deliver `appNotReady.png` screenshot.

Reply:
Otherwise, if let's say `detox.init()` fails with timeout in the upper `beforeAll`, users will be left scratching their heads with almost empty artifacts folder, which is not something I really wish. That's why I am adding it.